api/tests/test_cloud_record_summary.py

```diff
@@ -127,3 +250,75 @@ def test_filter_cursor(self):
     def tearDown(self):
         """Delete any messages under QPATH and re-enable logging.INFO."""
         logging.disable(logging.NOTSET)
+
+    def _populate_database(self, database):
```

Comment:
done

(sorry, when I said I had done all these at the bottom I was clearly lying, I am pretty sure they have all now been fixed or commented on)